BigQuery/src/BigQueryClient.php

```diff
@@ -682,4 +682,22 @@ public function timestamp(\DateTimeInterface $value)
     {
         return new Timestamp($value);
     }
+
+    /**
+     * Get a service account for the KMS integration.
+     *
+     * Example:
+     * ```
+     * $serviceAccount = $bigQuery->getServiceAccount();
+     * ```
+     * @param array $options [optional] Please see the
```

Comment:
`@param array $options [optional] Configuration options.` is okay :). The link here goes to the information for a query job.